production/helm/loki/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Entries should include a reference to the pull request that introduced the chang
 
 [//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries bellow this line.)
 
+## 6.20.1
+- [BUGFIX] Optionally allow prefixing in Ingester rollout-group labels with user defined string
```

Comment:
Please do not add your own version. See the comment above, we'll automatically add the version for all unreleased features in the weekly release.